src/models/mod.rs

```diff
@@ -125,7 +125,7 @@ pub(crate) async fn start_after_interruption(
     pool: &sqlx::Pool<sqlx::Postgres>,
 ) -> anyhow::Result<u64> {
     let query = "SELECT block_height
-                        FROM blocks
+                        FROM near_balance_events
                         ORDER BY block_timestamp desc
                         LIMIT 1";
```

Comment:
I can't put the comment on line 136 so I leave it here

If we invoke it the first time, the DB is empty, and we have 0 as the default value. It makes no sense.

I suggest we get rid of default value, throwing the error something like "`start_block_height` should be provided at the first launch"

Reply:
Could you please do the same in events project when you finish with this review?